shapes/neurosciencegraph/datashapes/atlas/geneexpressionvolumetricdatalayer/schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": [
+    "https://incf.github.io/neuroshapes/contexts/schema.json",
+    {
+      "this": "https://neuroshapes.org/dash/geneexpressionvolumetricdatalayer/shapes/"
+    }
+  ],
+  "@id": "https://neuroshapes.org/dash/geneexpressionvolumetricdatalayer",
+  "@type": "nxv:Schema",
+  "imports": [
+    "https://neuroshapes.org/dash/volumetricdatalayer"
+  ],
+  "prov:wasDerivedFrom": "https://github.com/INCF/neuroshapes/blob/v0.3.15/modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/geneexpressionvolumetricdatalayer/v1.0.0.json",
```

Comment:
There were no geneexpressionvolumetricdatalayer schema in Neuroshapes v0 . I suggest ot remove the following reference: 

https://github.com/INCF/neuroshapes/blob/v0.3.15/modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/geneexpressionvolumetricdatalayer/v1.0.0.json